velox/dwio/common/ScanSpec.cpp

```diff
@@ -202,6 +202,50 @@ bool testIntFilter(
   return true;
 }
 
+template <typename T>
+bool testDecimalFilter(
```

Comment:
Thanks for the suggestion. Combined it with testIntFilter. Since `testIntFilter<int64_t>` and `testIntFilter<int128_t>` are used only once, changed to call them directly.